.gitignore

```diff
@@ -1,6 +1,6 @@
 #Pycharm
 .idea
-
+.DS_Store
```

Comment:
Why did you remove the .DS_Store line from .gitignore?

Reply:
I added .DS_Store to .gitignore, not removed it.

Reply:
Ah, yes. I am blind. Good point!